build_resolvers/pubspec.yaml

```
@@ -1,13 +1,13 @@
 name: build_resolvers
-version: 2.0.10-dev
+version: 2.0.10
 description: Resolve Dart code in a Builder
 repository: https://github.com/dart-lang/build/tree/master/build_resolvers
 
 environment:
   sdk: ">=2.17.0 <3.0.0"
 
 dependencies:
-  analyzer: ^4.6.0
+  analyzer: '>=4.4.0 <6.0.0'
```

Is this safe to downgrade the min version?

```suggestion
  analyzer: '>=4.6.0 <6.0.0'
```